lib/game.rb

```diff
@@ -6,32 +6,53 @@
 require_relative "./game_checker"
 
 class Game
-  attr_reader :display, :board, :player_1
-  def initialize(display, presenter, board, players)
-    @display = display
-    @presenter = presenter
-    @board = board
-    @player_1 = players[0]
-    @player_2 = players[1]
+  attr_reader :display, :board, :player_1, :player_2, :active_player, :game_checker
+  def initialize(config_object)
+    @display = config_object[:display]
+    @presenter = config_object[:presenter]
+    @board = config_object[:board]
+    @player_1 = config_object[:player_1]
+    @player_2 = config_object[:player_2]
+    @game_checker = config_object[:game_checker]
 
     @active_player = @player_1
   end
 
   def start
     @display.output(WELCOME)
+    print_board
   end
 
   def play
-    print_board
-    @board.place_mark(@player_1.mark, @player_1.get_input(@display))
-    print_board
-    @board.place_mark(@player_2.mark, @player_2.get_input(@display))
-    print_board
+    loop do
+      @board.place_mark(@active_player.mark, @active_player.get_input(@display))
+      print_board
+      switch_player
+      if @game_checker.game_over?(@board, @player_1, @player_2)
+        @display.output(GAME_OVER)
+        break
+      end
+    end
+  end
+
+  def switch_player
+    @active_player = @active_player == @player_1 ? @player_2 : @player_1
+  end
+
+  def end_game
+    if @game_checker.tie?(@board, @player_1, @player_2)
```

Comment:
These are non-blocking comments.  You can just pass an outcome to the presenter, and the presenter decides what to say (ex. if a tie, then presenter outputs TIE).